webapp/src/colors.tsx

```diff
@@ -306,6 +317,11 @@ export const colors = {
       mainText: '#BEF4E9',
       linkText: '#dddddd',
     },
+    emailNotVerifiedBanner: {
+      background: '#822343',
+      mainText: '#FDECF2',
+      titleText: '#F6A7C2',
+    },
```

Comment:
This way of defining colors is deprecated, we now use tokens directly exported from figma (in `figmaTheme.ts`). If you check what colors are used in figma, the same colors should be available in `theme.palette.tokens...`. If there are new colors, which are not in the file, you need to use our figma plugin to update the palette https://github.com/tolgee/figma-color-extractor (I can help you with that if needed).

Reply:
aha, ok